bin/emqx

```diff
@@ -652,6 +658,16 @@ if [ -z "$COOKIE" ]; then
     die "Please set node.cookie in $EMQX_ETC_DIR/emqx.conf or override from environment variable EMQX_NODE__COOKIE"
 fi
 
+## check if OTP version has mnesia_hook patch; if not, fallback to
+## using Mnesia DB backend.
+if [ "$(get_config_value 'db.backend')" = "rlog" ]; then
```

Comment:
check only if `$IS_BOOT_COMMAND` is set to `'yes'`

Reply:
no need to check when the variables are already set.
i.e.
`[ "${EMQX_DB__BACKEND:-}" = 'mnesia' ] && [ "${EMQX_DB__ROLE:-}" = 'core' ]`